packages/tools/src/drawingSvg/drawEllipse.ts

```diff
@@ -9,8 +9,7 @@ function drawEllipse(
   svgDrawingHelper: SVGDrawingHelper,
   annotationUID: string,
   ellipseUID: string,
-  corner1: Types.Point2,
-  corner2: Types.Point2,
+  canvasCoordinates: [Types.Point2, Types.Point2, Types.Point2, Types.Point2],
```

Comment:
Is there any way that we don't change the signature of this function to avoid causing breaking changes?

Reply:
Thank you for your feedback. 
It's necessity of having more than two corner points to accurately determine the ellipse rotation angle. 
In my opinion, passing all four coordinates is a logical solution.
But I appreciate your flexibility and willingness to maintain the current state of the `drawEllipse` function. 
For me, there are two possible ways to address this issue:

1. Implement `drawEllipseByCoordinates`: This option provides a specific and precise solution for users who require accurate ellipse rendering. Keeping `drawEllipse` unchanged ensures backward compatibility, allowing users to choose the method that best fits their needs.

2. Pass Rotation Angle with `options` object with a default value of 0. This maintains simplicity for users who don't require rotation while offering enhanced functionality for those who do.

I think I will stick to the first option, as it seems to be more developer-friendly.

Reply:
@sedghi changes was pushed, ping me if you prefer to stick to second option with `options` object